src/WS.cpp

```diff
@@ -1217,26 +1351,63 @@ signed char WS::stream2_callback(struct lejp_ctx *ctx, char reason)
                 {
                 }
             }
-            append_message(
-                "%d", cfg->get<int>(u_ctx->path));
+            append_session_msg(
+                u_ctx->message, "%d", cfg->get<int>(u_ctx->path));
             break;
+        /*
         case PNT_STREAM2_JPEG_REFRESH:
```

Comment:
Do we need this commented out code path here still? It's probably out of date now.

Reply:
will be done in future, if i'am really sure we do not need jpeg_refresh.
This value now is replaced by fps.